colossalai/tensor/module_handlers/linear.py

```diff
@@ -0,0 +1,34 @@
+from .colo_module_handler import ColoModuleHandler
+from colossalai.tensor import ComputePattern, distspec
+from colossalai.core import global_context as gpc
+from colossalai.context.parallel_mode import ParallelMode
+
+class ColoLinear(ColoModuleHandler):
+    def __init__(self):
+        super(ColoLinear, self).__init__()
+        self._register_shard_params(['weight', 'bias'])
+        self._set_TP1D()
```

Comment:
why set TP1D? ColoLinear can not use DP?

Reply:
DP is a default pattern now. It's not in ComputePattern.